masschallenge/django-accelerator
# [AC-8424] Investigate why migration command fails to apply migrations through accelerate instance on the local environment [Proof of concept PR]

## [AC-8424](https://masschallenge.atlassian.net/browse/AC-8424)

**Changes introduced**
- Add migration to alter model options: Update swappable option to None. 
It's worth mentioning that the options dict is an exact duplicate of what we have on 0001_initial.py with the swappable option set to None.

**Testing**
[local env]
- checkout to this branch
- run migrations through accelerate. 
- Confirm that migration(0028_mentorprogramofficehour_test_field.py) is applied successfully 

[test5]
- ssh and confirm that migrations are applied successfully on test5

Updating the swappable option to None allows migrations to be run and applied successfully through all stacks
